docs/versions.adoc

```diff
@@ -23,6 +23,6 @@ software.
 | 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z, 1.0.0-preZ, 1.0.0 | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
 | 0.1.1 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z, 1.0.0-preZ, 1.0.0 | 1.5.z |  5.2.z | 3.2.z | 1.6.z | 2.10.z
 | 0.2.0 | 1.2.z, 2.0.z, 2.1.z, 2.2.z, 3.0.z, 3.11.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z, 1.3.z | 0.9.z, 1.0.0-preZ, 1.0.0 | 1.5-1.7.z, 2.3-2.4.z, 5.y, 6.y |  5.2-5.5.z, 6.2-6.6.z, 7.y | 3.2.z | 1.6.z | 2.10.z
-| 0.3.0 | 1.2.z, 2.0.z, 2.1.z, 2.2.z, 3.0.z, 3.11.z | 1.0.z, 1.1.z, 1.2.z, 1.3.z | 0.9.z, 1.0.0-preZ, 1.0.0 | 1.5-1.7.z, 2.3-2.4.z, 5.y, 6.y |  5.2-5.5.z, 6.2-6.6.z, 7.y | 3.2.z | 1.6.z | 2.10.z
+| 0.3.0 | 1.2.z, 2.0.z, 2.1.z, 2.2.z, 3.0.z, 3.11.z | 1.0.z, 1.1.z, 1.2.z, 1.3.z | 1.0.0, 1.1.0, 1.1.2, 1.2.0, 1.3.0 | 1.5-1.7.z, 2.3-2.4.z, 5.y, 6.y |  5.2-5.5.z, 6.2-6.6.z, 7.y | 3.2.z | 1.6.z | 2.10.z
```

Comment:
Could it be `1.y` instead?

Reply:
@sjudeng – I considered that, but there was a specific issue in the [Cloud Bigtable 1.1.1 release](https://github.com/GoogleCloudPlatform/cloud-bigtable-client/releases/tag/bigtable-client-1.1.1) that I wanted to exclude it from the list here.

For HBase, the list of compatible versions is `1.0.z, 1.1.z, 1.2.z, 1.3.z` which was the pattern I also considered using, but that would include 1.1.1 in the list as well.